.travis.yml

```diff
@@ -1,14 +1,34 @@
 language: python
 python:
+  - "3.4"
```

Comment:
I don't know why these weren't in ascending order to start with, but can we change to that, please?

At least then it'll match the Django version ordering. We can also drop official Python 2.6 support probably.